### ISSUE TITLE
crossmedia loss只下降了一次.

### ISSUE THREAD

**Issue by yuandaxing** [alibaba/x-deeplearning#195]
crossmedia这个例子数据格式有很多错误, fix了以后,  data.txt中只留一条数据, 发现loss只下降了一次, 然后就无法下降了, 打印如下:
loss:  [array(0.6933602, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]
loss:  [array(0.6931472, dtype=float32)]

1. 能不能提供一个目前在官方镜像可以跑的crossmedia

**Comment by UanonymousU** [comment]
在分布式训练从 ali-userbehavior 数据集中截取 120w 数据时，遇到 loss 一直在 0.65 左右徘徊，不下降的现象：（起了一个 scheduler（8 核 CPU 被打满），两个 ps（这两个 docker 在同一物理机上，8 核 CPU 被打满），四个 worker）

> ... ...
> #### loss:[0.6520809]
> #### loss:[0.651621]
> #### loss:[0.65042067]
> #### loss:[0.6509485]
> #### loss:[0.6542247]
> #### loss:[0.651094]
> #### loss:[0.65301454]
> #### loss:[0.65175164]
> #### loss:[0.6507039]
> #### loss:[0.6516748]
> #### loss:[0.65354496]
> #### loss:[0.65325475]
> #### loss:[0.6509405]
> save checkpoint at global_step[10000], ckpt version[ckpt-...............10000]
> #### loss:[0.65081173]
> #### loss:[0.6511547]
> #### loss:[0.6519633]
> #### loss:[0.6520231]
> #### loss:[0.65135163]
> #### loss:[0.65134734]
> #### loss:[0.6528831]
> #### loss:[0.6521726]
> #### loss:[0.65207523]
> #### loss:[0.65380156]
> #### loss:[0.6517576]
> #### loss:[0.65218675]
> #### loss:[0.6503474]
> #### loss:[0.6532894]
> ... ...

附加 tdm.json配置：

> {
>     "train_rawdata_url": "hdfs://ip:port/user/rawdata/mock_train.csv",
>     "test_rawdata_url": "hdfs://ip:port/user/rawdata/mock_test.csv",
>     "data_dir": "data",
>     "tree_filename": "userbehavoir_tree.pb",
>     "train_sample": "userbehavoir_train_sample.dat",
>     "test_sample": "userbehavoir_test_sample.dat",
>     "stat_file": "userbehavoir_stat.dat",
>     "feature_conf": "userbehavoir_fc.json",
>     "train_sample_segments": 400,
>     "seq_len": 70,
>     "min_seq_len": 6,
>     "parall": 16,
>     "train_id_label": "train_unit_id",
>     "test_id_label": "test_unit_id",
>     "upload_url": "hdfs://ip:port/user/dist_tree/upload/tree_data",
>     "epocs": 1,
>     "model_url": "hdfs://ip:port/user/dist_tree/emb_converted",
>     "item_id_file": "leaf.id",
> 
>     "train_mode": "sync",
>     "learning_rate": 0.001,
>     "train_batch_size": 30000,
>     "train_epochs": 1,
>     "train_threads": 16,
>     "save_checkpoint_interval": 1000,
>     "predict_batch_size": 10000,
>     "predict_epochs": 1,
>     "predict_threads": 1,
>     "saver_ckpt": "",
> 
>     "tree_store_config": "size=10000000",
>     "use_hdfs_tree": 1,
>     "tree_pb": "data/ub_tree.pb",
>     "predict_io_pause_num": 10000,
>     "tdmop_debug": "false",
>     "tdmop_layer_counts": "0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,17,19,22,25,30,76,200",
> 
>     "_end_": 0
> }

而在单机上训练时（tdm.json 与分布式配置相同），跑了几个 session 后损失就下降到 0.4 以下了。请问可能是哪儿出了问题呢？另外， #207 中描述的现象依然存在：worker0 在完成了 put item_embed 之后，其他 worker 还在继续跑 session。
@songyue1104 @yiling-dc @largestone1982 感谢！
